### PR TITLE
add mesos http and tcp health check

### PR DESCRIPTION
**Pull request by pwzgorilla**

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -88,6 +88,14 @@ func main() {
 			Name:  "enable-dns-proxy",
 			Usage: "enable dns proxy or not",
 		},
+		cli.BoolFlag{
+			Name:  "enable-local-healthcheck",
+			Usage: "Enable local healt check",
```
Comment (cmingxu) [inline main.go]: health

File: main.go [hunk 2]
```diff
@@ -88,6 +88,14 @@ func main() {
...
+		},
+		cli.BoolFlag{
+			Name:  "disable-raft",
```
Comment (cmingxu) [inline main.go]: how about add flag `standalone`, which means doesn't run swan cluster mode

**Comment by cmingxu** [comment]
LGTM

File: main.go (hunk 2 above)
Comment (pwzgorilla) [inline main.go]: good

**Pull request merged**